screens/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
+import Form from 'react-native-form';
+import { useState } from 'react';
+
+export const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const submitForm = () => {
+    // query firebase
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.h1_heading}>
+        Welcome back!
+        <br />
+        Please login.
+      </Text>
+
+      <Form style={styles.form_container}>
```

Comment:
Getting error: Property 'ref' is missing in type '{ children: Element[]; style: { width: string; height: string; }; }' but required in type 'Readonly<FormProps>'. Be sure that you're including all necessary props for these components!